### PR TITLE
Price updates

### LINKED ISSUE #2077: Build UA price pages
Breif: https://docs.google.com/a/canonical.com/document/d/1QA4aiDxBjDsqpa0s5iHocmChxTrnQ1ibrSDjCXIw2Ls/edit?usp=sharing

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated the price information of various cloud products.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/cloud/plans-and-pricing](http://0.0.0.0:8001/cloud/plans-and-pricing)
- See that the updates in the [copydoc](https://docs.google.com/document/d/1yZ7wClcmxGCqMyyyBJLO66Z3YlpT-4soWLFmJ6uXeMM/edit) are all addressed
- Go to: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- See that the included shared portions updated for the other page look good here too

## Issue / Card

Fixes #2077

### CODE REVIEW

**Comment by joanasa89** [comment]
Looks ok to me, just waiting for Frankie's final review to change my label.

File: templates/cloud/plans-and-pricing.html [hunk 1]
```diff
@@ -13,24 +13,55 @@
...
+        <tbody>
+          <tr>
+            <th scope="row"><a href="/server/maas">MAAS</a></th>
+            <td>FREE</td>
+            <td>From $5 per server per month</td>
+          </tr>
+          <tr>
+            <th scope="row"><a href="/cloud/juju">Juju</a></th>
+            <td>FREE</td>
+            <td>Find out more at jujucharms.com </td>
```
Comment (anthonydillon) [inline templates/cloud/plans-and-pricing.html]: Make this an external link please.

File: templates/cloud/plans-and-pricing.html [hunk 2]
```diff
@@ -13,24 +13,55 @@
...
+          <tr>
+            <th scope="row"><a href="/support">Landscape</a></th>
+            <td>
+              Free up to ten machines (on-premises)<br />
+              or<br />
+              $100 free credit for 60 days (SaaS)
+            </td>
+            <td>
+              SaaS included as part of Ubuntu Advantage<br />
+              $2,000 per year - Landscape on-premises<br />
```
Comment (anthonydillon) [inline templates/cloud/plans-and-pricing.html]: Encode the dash please.

File: templates/shared/_ubuntu_advantage_cloud_per_node.html [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
-  <li class="p-list__item is-ticked">Landscape systems management for hosts and Ubuntu guests</li>
-  <li class="p-list__item is-ticked">3 terabytes of Ceph or Swift based usable storage</li>
-</ul>
+<div class="row u-equal-height">
+  <div class="p-card col-6">
+    <div class="p-card__header--bordered">
+      <h3 class="p-heading--five">Ubuntu Advantage support for OpenStack</h3>
+      <p class="p-heading--two">$1,500 <span class="p-heading--six">per year</span></p>
+    </div>
+    <p class="p-card__content"></p>
```
Comment (anthonydillon) [inline templates/shared/_ubuntu_advantage_cloud_per_node.html]: Can you remove this  empty element please

File: templates/shared/_ubuntu_advantage_cloud_per_node.html [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
+      <li class="p-list__item is-ticked">Landscape systems management for hosts and Ubuntu guests</li>
+      <li class="p-list__item is-ticked">2 terabytes of Ceph or Swift based usable storage</li>
+    </ul>
+  </div>
+  <div class="p-card col-6">
+    <div class="p-card__header--bordered">
+      <h3 class="p-heading--five">BootStack &mdash; Fully managed OpenStack</h3>
+      <p class="p-heading--two">$15 <span class="p-heading--six">a day per node</span></p>
+    </div>
+    <p class="p-card__content"></p>
```
Comment (anthonydillon) [inline templates/shared/_ubuntu_advantage_cloud_per_node.html]: Remove empty element please

File: static/sass/styles.scss [hunk 5]
```diff
@@ -313,3 +313,9 @@ sup {
     margin-bottom: 1rem;
   }
 }
+
+/// XXX Give card headers an underlined option
+/// This will need replacing when the latest vanilla card changes land
+.p-card__header--bordered {
```
Comment (deadlight) [inline static/sass/styles.scss]: I think that I'll clean this up better. The problem is existing card headers on the site not having borders.

**Comment by anthonydillon** [comment]
QA issues:
- The intro text is in one paragraph in the copy doc.
- The link to Landscape in the cloud tools section should be to: /server/management you have /support
- In the Service provider section there is a Buy Ubuntu Advantage link which is not in the copy doc
- I think the "BootStack fully managed OpenStack Cloud" section should be in a `col-9` to stop the ophan cloud in the title
- The link `Schedule a BootStack demo` should be to /cloud/contact-us?product=managed-cloud not /cloud/contact-us?product=cloud-plans-and-pricing
- The logo section in the "Ubuntu on public clouds" section does not match the copy doc

**Comment by deadlight** [comment]
>The intro text is in one paragraph in the copy doc.
Updated the copydoc.

>The link to Landscape in the cloud tools section should be to: /server/management you have /support
Updated

>In the Service provider section there is a Buy Ubuntu Advantage link which is not in the copy doc
Added to copydoc

>I think the "BootStack fully managed OpenStack Cloud" section should be in a col-9 to stop the ophan cloud in the title
Done

>The link Schedule a BootStack demo should be to /cloud/contact-us?product=managed-cloud not /cloud/contact-us?product=cloud-plans-and-pricing
Updated in code and copydoc

>The logo section in the "Ubuntu on public clouds" section does not match the copy doc
The one in the code is correct. I'll update one of the two copydocs that use it and point the other one to it, to avoid duplicates getting out of sync.

**Comment by deadlight** [comment]
@anthonydillon ready for re-review

@frankiegranato ready for UX review

**Comment by anthonydillon** [comment]
Thanks @deadlight

**Comment by deadlight** [comment]
@frankiegranato we're going live with this but please do a UX review when you're back.

**Pull request merged**